### PR TITLE
Use quartz allocation for its CI jobs

### PR DESCRIPTION
**Pull request by kennyweiss**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ stages:
    #BUILD + TEST
     - echo -e "section_start:$(date +%s):build_and_test\r\e[0K
       Build and test ${CI_PROJECT_NAME}"
-    - ${EXEC_PREFIX} .gitlab/build_and_test.sh
+    - ${ALLOC_COMMAND} .gitlab/build_and_test.sh
```
Comment (kennyweiss) [inline .gitlab-ci.yml:25]: See:
https://github.com/LLNL/blt/blob/68c98b7306e0b1f19cb383ac6af210d6f75627fc/.gitlab/build_quartz.yml#L39-L43

**Comment by kennyweiss** [comment]
With this change, the `LLNL.blt` pipeline finished in under a minute and a half  vs. ~5-15 minutes previously

**Comment by kennyweiss** [comment]
Looks like the lassen build/tests haven't been running for a bit in our `LLNL.blt` CI. 
Trying to fix that as well...

**Comment by kennyweiss** [comment]
Update: Looks like I fixed the Lassen tests -- CI came in at a respectable 2:49 w/ 169 tests!

File: .gitlab-ci.yml (hunk 1 above)
Comment (white238) [reply .gitlab-ci.yml:25]: whoops my bad

**Review by white238 — APPROVED** [review]

**Pull request merged**